packages/ketcher-polymer-editor-react/src/Editor.tsx

```diff
@@ -111,6 +101,9 @@ function MenuComponent() {
 
   const menuItemChanged = (name) => {
     dispatch(selectTool(name))
+    if (modalComponentList[name]) {
+      dispatch(openModal(name))
+    }
   }
```

Comment:
I don't think it's a good idea. For one, it sets active tool to 'Open', but there's no such tool. We should have some separation of this logic, and it probably should be handled inside of `Editor`.